dash/App/Tokens/AddToken/index.js

```diff
@@ -209,11 +214,11 @@ class AddTokenFormScreen extends Component {
               Add New Token
             </div>
             <div className='newTokenChainSelectTitle'>
-              <div className='newTokenChainAddress'>
+              <h2 className='newTokenChainAddress'>
```

Comment:
don't think we should change this element just to make the test work, this could have impacts on styling either now or down the road and just semantically changes the document. if anything if we want to consider this `div` as a heading then we should just add the role explicitly

Reply:
yeah realised as you were typing this and fixed it